src/peft/tuners/lora/config.py

```diff
@@ -117,6 +117,26 @@ class LoraConfig(PeftConfig):
             )
         },
     )
+    megatron_config: Optional[dict] = field(
+        default_factory=dict,
```

Comment:
Since this is supposed to be an optional argument, how about making the default `None`?

Reply:
> Since this is supposed to be an optional argument, how about making the default `None`?

ok.